src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -1248,8 +1240,8 @@ class SpatialPooler : public Serializable
   bool wrapAround_;
   UInt updatePeriod_;
 
-  Real synPermMin_;
-  Real synPermMax_;
+  Real synPermMin_ = 0.0;
+  Real synPermMax_ = 1.0; //TODO set in initialize(), somehow OSX does not set that?!
```

Comment:
@dkeeney can you explain this? It's later set in `initialize()`, Linux OK, but OSX fails w/o this